webpack.config.js

```diff
@@ -43,7 +44,12 @@ let conf = {
   plugins: [
     isProfile ? new BundleAnalyzerPlugin() : null,
 
-    new webpack.EnvironmentPlugin(envVars),
+    new webpack.EnvironmentPlugin({
+      NODE_ENV: 'development',
```

Comment:
Shouldn't this be left to the actual env value?

Reply:
Yeah it should